### DISCUSSION TITLE
strange results when running VSG vsgViewer.cpp in Windows 10

### DISCUSSION THREAD

**Discussion by RKiserAL** [vsg-dev/VulkanSceneGraph#777] [Q&A]
Hello All

I just downloaded the latest version of VSG (Version 1.0.5) and saw some strange results when trying out the example: vsgViewer.cpp

When I ran it pointing to lz.vsgt, all was fine and I saw a display of topography.

When I ran it pointing to teapot.vsgt, I got the following error messages:

ERROR: Unable to match userObjects got NumUserObjects instead.
ERROR: Unable to match children got 1 instead.
ERROR: Unable to match matrix got Key instead.
ERROR: Unable to match subgraphRequiresLocalFrustum got "batch" instead.
ERROR: Unable to match } got Object instead.

Just a blue background was displayed.

Both of these .vsgt files were obtained from the data sub-directory of the vsgExamples directory.

Is this some kind of pilot error, i.e., I am specifying a file incompatible with vsgViewer, or something else?

I am a complete newbie with VSG, so I'm suspecting pilot error.  I do, however, have quite a bit of experience with OSG.

For what it's worth (if anything):
Windows 10 x64
Visual Studio 2022 Community 17.5.3
Vulkan 1.3.243
NVIDIA GeForce GTX 1080

Any ideas?

Thanks in advance for any help,

Bob Kiser

**Comment by robertosfield** [comment]
Hi Bob,

From the look of the error it sounds like the vsgExamples/data that you are using is an early incarnation, prior to the VSG-1.0 release and contains no longer valid .vsgt files.

You could try updating to vsgExamples master or vsgExamples-1.0.x.

Cheers,
Robert.

**Comment by RKiserAL** [comment]
Yes, that did it.  Thank you for your quick response.

How I got an old version I don't know as I thought I got them from the latest GitHub repository.  So, my screw-up.

**Comment by robertosfield** [comment]
On Tue, 11 Apr 2023 at 14:36, RKiserAL ***@***.***> wrote:

> Yes, that did it. Thank you for your quick response.
>
> How I got an old version I don't know as I thought I got them from the
> latest GitHub repository. So, my screw-up.
>

Looking at the history of teapot.vsgt shows the update from last June:

https://github.com/vsg-dev/vsgExamples/commit/27fa5eb11abfa7065f2638a9c054bd85a27835d6

So my best guess is that you must have had a clone the vsgExamples repo
before last summer and still had this hanging around on your VSG_FILE_PATH.

 Message ID: <vsg-dev/VulkanSceneGraph/repo-discussions/777/comments/5581005
@github.com>

>
